src/polycubed/src/server/Resources/Endpoint/LeafResource.cpp

```diff
@@ -113,6 +113,13 @@ void LeafResource::CreateReplaceUpdate(const Pistache::Rest::Request &request,
     auto op = Operation::kUpdate;
     auto resp = WriteValue(cube_name, jbody, keys, op);
     errors.push_back(resp);
+    if (resp.error_tag == ErrorTag::kOk ||
```

Comment:
Not clear to me what you're doing. Please add a comment.
The same for L178 below.